lib/msf/core/db_manager/host.rb

```diff
@@ -157,8 +157,6 @@ def report_host(opts)
 
     if not addr.kind_of? ::Mdm::Host
       addr = normalize_host(addr)
-      addr, scope = addr.split('%', 2)
-      opts[:scope] = scope if scope
```

Comment:
we do that in normalize host and then don't do anything else with it
https://github.com/rapid7/metasploit-framework/blob/staging/rails-4.0/lib/msf/core/db_manager/host.rb#L88-L92